packages/remediations/src/RemediationWizard/RemediationWizard.js

```diff
@@ -115,6 +129,7 @@ const RemediationWizard = ({
                 FormTemplate={(props) => <Pf4FormTemplate {...props} showFormControls={false} />}
                 initialValues={{
                     [HAS_MULTIPLES]: !!state.resolutions?.find(r => r.resolutions.length > 1),
+                    [SYSTEMS]: undefined,
```

Comment:
I've noticed that there is a lot of accessors with fallback value to `[]` or length to `0`. Is it possible to use an empty array as an initial value or are there some difficulties with it?

Reply:
@Hyperkid123  Undefined value is used for the automatic selection of all items on the initial load of the step. It needs to be different from empty array to avoid this selection happening when loading data on paginating. I can't think of a better way to solve this yet.